src/trinerdi/template.cpp

```diff
@@ -0,0 +1,15 @@
+#include <bits/stdc++.h>
+
+using namespace std;
+
+typedef long long ll;
+typedef long double ld;
+#define rep(i, a, n) for (int i = (a); i < (n); i++)
+#define per(i, a, n) for (int i = (n) - 1; i >= (a); i--)
+
+int n;
+
+int main(void) {
+	ios_base::sync_with_stdio(false);
```

Comment:
nitpicking, but `0` would suffice over `false` (if we're going for terseness)

Reply:
That's right, fixed.